tests/model_execution/err_kparse_reference/custom_gul_summarycalc_1_output_1_partition.0.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+SCRIPT=$(readlink -f "$0") && cd $(dirname "$SCRIPT")
+
+# --- Script Init ---
+set -euET -o pipefail
+shopt -s inherit_errexit 2>/dev/null || echo "WARNING: Unable to set inherit_errexit. Possibly unsupported by this shell, Subprocess failures may not be detected."
+
+LOG_DIR=log
+mkdir -p $LOG_DIR
+rm -R -f $LOG_DIR/*
+
+
+touch $LOG_DIR/stderror.err
+ktools_monitor.sh $$ $LOG_DIR & pid0=$!
+
+exit_handler(){
+   exit_code=$?
+
+   # disable handler
+   trap - QUIT HUP INT KILL TERM ERR EXIT
+
+   kill -9 $pid0 2> /dev/null
+   if [ "$exit_code" -gt 0 ]; then
+       # Error - run process clean up
+       echo 'Ktools Run Error - exitcode='$exit_code
+
+       set +x
+       group_pid=$(ps -p $$ -o pgid --no-headers)
+       sess_pid=$(ps -p $$ -o sess --no-headers)
+       script_pid=$$
+       printf "Script PID:%d, GPID:%s, SPID:%d
+" $script_pid $group_pid $sess_pid >> $LOG_DIR/killout.txt
+
+       ps -jf f -g $sess_pid > $LOG_DIR/subprocess_list
+       PIDS_KILL=$(pgrep -a --pgroup $group_pid | awk 'BEGIN { FS = "[ \t\n]+" }{ if ($1 >= '$script_pid') print}' | grep -v celery | egrep -v *\\.log$  | egrep -v *\\.sh$ | sort -n -r)
+       echo "$PIDS_KILL" >> $LOG_DIR/killout.txt
+       kill -9 $(echo "$PIDS_KILL" | awk 'BEGIN { FS = "[ \t\n]+" }{ print $1 }') 2>/dev/null
+       exit $exit_code
+   else
+       # script successful
+       exit 0
+   fi
+}
+trap exit_handler QUIT HUP INT KILL TERM ERR EXIT
+
+check_complete(){
+    set +e
+    proc_list="eve getmodel gulcalc fmcalc summarycalc eltcalc aalcalc leccalc pltcalc ordleccalc"
+    has_error=0
+    for p in $proc_list; do
+        started=$(find log -name "$p*.log" | wc -l)
+        finished=$(find log -name "$p*.log" -exec grep -l "finish" {} + | wc -l)
+        if [ "$finished" -lt "$started" ]; then
+            echo "[ERROR] $p - $((started-finished)) processes lost"
+            has_error=1
+        elif [ "$started" -gt 0 ]; then
+            echo "[OK] $p"
+        fi
+    done
+
+    started=$( cat log/gul_stderror.err |  grep "Starting custom gulcalc command" | wc -l)
+    finished=$( cat log/gul_stderror.err |  grep "Custom gulcalc command finished" | wc -l)
+    if [ "$finished" -lt "$started" ]; then
+        echo "[ERROR] gulcalc - $((started-finished)) processes lost"
+        has_error=1 
+    elif [ "$started" -gt 0 ]; then
+        echo "[OK] gulcalc"
```

Comment:
As a bonus, this could show the actual name of the custom gulcalc process instead of the generic `gulcalc`.